src/commands/build.rs

```diff
@@ -90,7 +159,9 @@ async fn process_ftd(
             .as_bytes(),
     )
     .await?;
-    println!("Generated {}", file_rel_path.as_str(),);
+    if print_log {
+        println!("Generated {}", file_rel_path.as_str(),);
```

Comment:
Do not print the path of generated file, print the path of the file that was processed.